ansible-scylla-node/templates/scylla.yaml.j2

```diff
@@ -21,7 +21,11 @@ commitlog_segment_size_in_mb: 32
 seed_provider:
     - class_name: org.apache.cassandra.locator.SimpleSeedProvider
       parameters:
+{% if temporary_seed is defined %}
```

Comment:
Why do you need this variable?
Why not to auto-select a seeds for a node being replaced as follows:

Names:
* Node A - node being replaced
* Node B - node that replaced Node A

1) Use the first node from the inventory which is not Node B (when you run this playbook inventory is supposed not to have Node A anymore) as a seeder for Node B.
2) After Node B bootstraps change its seeder list as follows:
  * If the original seeders list included "Node A" - replace Node A with Node B and use that list. 
  * Otherwise use seeders list as-is.
3) On every other node replace Node A with Node B in seeders list.

Reply:
I see that you kinda doing the above in following patches in the external playbook.
What I don't get why you need an additional variable here and can't simply use `scylla_seeds`?